runtime/bootstrap/index.ts

```diff
@@ -130,12 +133,103 @@ export async function executeThunk(
   context: Context,
   {params, formulaSpec}: {params: ParamValues<ParamDefs>; formulaSpec: FormulaSpecification},
 ): Promise<SyncFormulaResult<object> | PackFormulaResult> {
-  const resultRef = await context.evalClosure('return findAndExecutePackFunction($0, $1);', [params, formulaSpec], {
-    arguments: {copy: true},
-    result: {reference: true, promise: true},
-  });
+  const resultRef = await context.evalClosure(
+    'return coda.findAndExecutePackFunction($0, $1, pack.pack || pack.manifest, executionContext);',
+    [params, formulaSpec],
+    {
+      arguments: {copy: true},
+      result: {reference: true, promise: true},
+    },
+  );
 
   // And marshal out the results into a local copy of the isolate object reference.
   const localIsolateValue = await resultRef.copy();
   return localIsolateValue;
 }
+
+/**
+ * Injects the ExecutionContext object, including stubs for network calls, into the isolate.
+ */
+export async function injectExecutionContext({
+  context,
+  fetcher,
+  temporaryBlobStorage,
+  logger,
+  endpoint,
+  timezone,
+  invocationToken,
+  sync,
+}: {
+  context: Context;
+  fetcher: Fetcher;
+  temporaryBlobStorage: TemporaryBlobStorage;
+  logger: Logger;
+  endpoint?: string;
+  timezone: string;
+  invocationToken?: string;
+  sync?: Sync;
+}): Promise<void> {
+  // Inject all of the primitives into a global we'll access when we execute the pack function.
+  const executionContextPrimitives = {
+    fetcher: {},
+    temporaryBlobStorage: {},
+    logger: {},
+    endpoint,
+    invocationLocation: {
+      protocolAndHost: 'TBI',
```

Comment:
I have no idea what this is .. sounds like it needs some work? I'll add a TODO

Reply:
It should be easy, it's just the base url of the executing environment, e.g. `https://coda.io` or `https://dev.coda.io:8080`. It's used so that packs like cross-doc can properly work in every environment.

So we just need to inject the proper value from the server config when setting up this environment. For the CLI simulator, it can default to `https://coda.io`

Reply:
got it. I'll put https://coda.io here for now and leave a TODO to make sure this works in lambda.